gazel/diff.go

```diff
@@ -33,7 +34,8 @@ func Diff(left, right []byte) error {
 	}
 	rf.Close()
 
-	cmd := exec.Command("/usr/bin/diff", lf.Name(), rf.Name())
+	fmt.Println("Unexpected diff output:")
```

Comment:
I don't think this is the right place for this comment. If the diff is unexpected then whoever invokes gazel should say so.

The other change looks good to me.